docs/references/supported-signatures.mdx

```diff
@@ -0,0 +1,34 @@
+---
+keywords: [advanced, multi-chain, multichain, signatures, tecdsa, ecdsa, cryptography ]
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Supported signatures
+
+<MarkdownChipRow labels={["Reference"]} />
+
+## Currently supported signatures
+
+ICP currently supports the following cryptographic signature types:
+
+- [ECDSA](/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa): ECDSA is supported via the ICP's novel threshold ECDSA protocol. Threshold ECDSA uses private ECDSA keys that only exist as secret shares that are distributed across the nodes within the threshold ECDSA subnet. Signatures are generated by computing these secret shares without reconstructing the private key. [Learn more about t-ECDSA](/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa)
+
+- Schnorr: Schnorr signatures are supported for processing certain types of assets, such as Ordinals and BRC20 assets, and enable web-based smart contract wallets such as [Oisy wallet](https://oisy.com/).
```

Comment:
Are we sure this is accurate? I think we're working on Schnorr but it's not supported yet.

Reply:
Yes, far from that. They will be supported in the future.

Reply:
@dprats , Schnorr signatures are incorrectly listed as supported on this FAQ page: https://internetcomputer.org/docs/current/developer-docs/multi-chain/faq/signatures-faq#does-icp-support-schnorr-signatures